app/controllers/users.rb

```diff
@@ -0,0 +1,25 @@
+# REGISTER
+get '/users/new' do
+  erb :'users/new'
+end
+
+post '/users' do
+  @user = User.new(params[:user])
+  if @user.save
+    puts "User saved!"
+    redirect '/sessions/new'
+  else
+    @errors = @user.errors.full_messages
+    puts "User not saved!"
+    erb :'/users/new'
+  end
+end
+
+# DASHBOARD
+get '/users/:id' do
+  if logged_in?
```

Comment:
When we merged, we have a require_login method in our sessions helper. You can just require_login. No need for the if/else statement.